shared/constants/lbaitem.ts

```diff
@@ -0,0 +1,8 @@
+export enum LBA_ITEM_TYPE {
+  FORMATION = "formation",
+  RECRUTEURS_LBA = "recruteurs_lba",
+  OFFRES_EMPLOI_LBA = "offres_emploi_lba",
+  OFFRES_EMPLOI_PARTENAIRES = "offres_emploi_partenaires",
+}
+
+export const allLbaItemType: string[] = [...Object.values(LBA_ITEM_TYPE)]
```

Comment:
```suggestion
export const allLbaItemType: string[] = Object.values(LBA_ITEM_TYPE)
```
Pas besoin de faire une copie